### PR TITLE
Fix roman numeral

### LINKED ISSUE #1178: `øṘ` only works on uppercase letters

### LINKED ISSUE #1183: Make `øṘ` vectorize on LazyLists
It only does on non-lazy lists at the moment.

### LINKED ISSUE #1184: `øṘ` doesn't work for technically-invalid-but-valid roman numerals
for example `ICIV` should be `103`, but it returns 1: [Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJgSUNJVmAgw7jhuZgiLCIiLCIiXQ==)

### LINKED ISSUE #1185: `øṘ` doesn't work for values 4000 and above
It should be valid, just add tons of `M`s.

### PR DESCRIPTION
**Pull request by Steffan153**
Fixes #1183
Fixes #1184
Fixes #1185

Fixes #1178

### CODE REVIEW

**Review by Lyxal — APPROVED** [review]
LGTM

File: vyxal/elements.py [hunk 1]
```diff
@@ -4133,10 +4133,62 @@ def roman_numeral(lhs, ctx):
         "IV",
```
Comment (chunkybanana) [inline vyxal/elements.py:4133]: Clean up after yourself pls? (also it would be one line to fix #1178 while you're at it)

**Review by Lyxal — APPROVED** [review]
Still LGTM

**Review by chunkybanana — APPROVED** [review]
lgtm

File: vyxal/elements.py [hunk 2]
```diff
@@ -4133,24 +4133,77 @@ def roman_numeral(lhs, ctx):
...
-        for i, n in enumerate(nums):
+        for i, n in enumerate(big_nums):
             while lhs.startswith(n):
-                result += ints[i]
+                result += big_ints[i]
                 lhs = lhs[len(n) :]
         return result
-    elif vy_type(lhs) is list:
+    elif vy_type(lhs, simple=True) is list:
         return vectorise(roman_numeral, lhs, ctx=ctx)
```
Comment (chunkybanana) [inline vyxal/elements.py:4207]: for some reason, this is failing to vectorise, and I can't figure out why.

**Comment by chunkybanana** [comment]
tests passing, so merging

**Pull request merged**